ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -279,3 +279,56 @@ periodics:
         requests:
           cpu: 10m
     serviceAccountName: ci-operator
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: azure-test-conformance
```

Comment:
We need to start validating these names in CI. Or at least document the conventions we follow:
* `periodic-ci-{org}-{repo}-{branch}-{name}` for periodic jobs
eg. periodic-ci-openshift-openshift-azure-master-conformance
* `pull-ci-{org}-{repo}-{branch}-{name}` for PR (presubmit) jobs
eg. pull-ci-openshift-openshift-azure-master-conformance
* `branch-ci-{org}-{repo}-{branch}-{name}` for branch (postsubmit) jobs
eg. branch-ci-openshift-openshift-azure-master-conformance

Reply:
agreed. And potentially document all command available in our repos too.

Reply:
But this sound as separate story/issue

Reply:
At least follow the convention aka update the name to `periodic-ci-openshift-openshift-azure-conformance`.